charts/victoria-metrics-cluster/values.yaml

```diff
@@ -49,6 +49,19 @@ vmselect:
   extraLabels: {}
   env: []
 
+  # Readiness & Liveness probes
+  probe:
```

Comment:
could you please use all params from values file in template or remove them

btw what about vminsert and storage
do you think, can this be helpful for other components?